src/apps/lwaftr/lwutil.lua

```diff
@@ -66,10 +63,28 @@ local n_ethertype_ipv6 = constants.n_ethertype_ipv6
 function is_ipv6(pkt)
    return rd16(pkt.data + o_ethernet_ethertype) == n_ethertype_ipv6
 end
+
 function is_ipv4(pkt)
    return rd16(pkt.data + o_ethernet_ethertype) == n_ethertype_ipv4
 end
 
+function is_ipv6_fragment(pkt)
+   if not is_ipv6(pkt) then return false end
+   return pkt.data[ehs + constants.o_ipv6_next_header] ==
+   constants_ipv6_frag
```

Comment:
Would it be possible to move this line up? Makes it more readable and it's still within the 80-column wrap.